README.md

```diff
@@ -331,4 +336,8 @@
 
 ## Project Management
   
-  * [milestones](https://github.com/turbopape/milestones) 
+  * [milestones](https://github.com/turbopape/milestones)
+ 
+## Products
+
+  * [Metabase](http://www.metabase.com/)
```

Comment:
@RenanPalmeira ooh, seems I also lost the track of this :D 
metabase already listed https://github.com/razum2um/awesome-clojure/commit/042a6ad0a16b1ed51dca1fc231990bc16a9b02f9 sorry for misleading comment